### PR TITLE
Add minimal 3.x federation instructions (master)

### PR DESCRIPTION
**Pull request by alanking**
Open to suggestions on organizing the information.

### CODE REVIEW

File: docs/system_overview/federation.md [hunk 1]
```diff
@@ -161,3 +161,65 @@ In order to support commands such as `ils` and `ilsresc` across a 3.x to 4.0+ fe
...
+LocalZoneSID ZONEB_LOCAL_ZONE_SID
+RemoteZoneSID ZoneA-ZONE_KEY_FOR_ALL_OF_ZONE_A
+~~~
+The `LocalZoneSID` is equivalent to the `zone_key` shared in 4.x federated environments.
+The `RemoteZoneSID` is a string of the form `remoteZoneName-remoteZoneSID`. In this example, `remoteZoneName` is `ZoneA` and the `remoteZoneSID` is the `zone_key` for the remote zone. Multiple `RemoteZoneSID`s can be defined in `server.config` to simultaneously federate with multiple zones.
+
+Information such as the remote zone's hostname and port are stored in the catalog in 3.x rather than in the server's configuration.
+
+As 3.x is built and run-in-place by the zone maintainer(s), the following diff can be applied to require SIDs be configured on the local server:
+```diff
```
Comment (alanking) [inline docs/system_overview/federation.md:212]: I'm actually not sure if this is necessary for getting this to work. It does give occasion to explain how the RemoteZoneSID should be structured, though. Might be a good idea to leave as an optional step (once I confirm that it is indeed optional). Thoughts?

File: docs/system_overview/federation.md [hunk 2]
```diff
@@ -161,3 +161,65 @@ In order to support commands such as `ils` and `ilsresc` across a 3.x to 4.0+ fe
 
 The effect of turning this negotiation off is a lack of SSL encryption when talking with a 3.x zone.  All clients that connect to this 4.0+ zone will also need to disable the Advanced Negotiation in their own 'irods_environment.json' files.
 
+### How to enable federation between 3.x and 4.x:
+
+The steps required for enabling federation between a 3.x zone and a 4.x zone will be described through an example.
+
+Let ZoneA have the following properties:
```
Comment (trel) [inline docs/system_overview/federation.md:168]: A is 4.x and B is 3.x?  please make that clearer up top.

File: docs/system_overview/federation.md (hunk 1 above)
Comment (trel) [reply docs/system_overview/federation.md:212]: let's leave it, anything we/you learn is good to write down.

File: docs/system_overview/federation.md (hunk 2 above)
Comment (alanking) [reply docs/system_overview/federation.md]: I added explicit mentions of the versions for the two zones on these lines. Thanks!

**Comment by trel** [comment]
seems good to me - please create an issue in irods/irods for tracking in the release notes - and then add it here.

**Comment by alanking** [comment]
Done

**Pull request merged**